client/settings/views/services/group.js

```diff
@@ -32,14 +32,16 @@ const ShippingServiceGroup = ( props ) => {
 		errors,
 		predefinedPackages,
 	} = props;
-	const summary = summaryLabel( services );
 	const actionButton = (
 		<button className="foldable-card__action foldable-card__expand" type="button">
 			<span className="screen-reader-text">{ __( 'Expand Services' ) }</span>
 			<Gridicon icon="chevron-down" size={ 24 } />
 		</button>
 	);
-	const allChecked = _.every( services, ( service ) => service.enabled );
+	//filter out the services that have a disabled corresponding predefined package
+	const manageableServices = _.filter( services, ( service ) => ! service.predefined_package || predefinedPackages.includes( service.predefined_package ) );
```

Comment:
No need to use `lodash` if `services` is an array. Just use `Array.prototype.filter()`.